src/internal/m365/collection/groups/collection_test.go

```diff
@@ -0,0 +1,159 @@
+package groups
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/alcionai/clues"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/alcionai/corso/src/internal/data"
+	"github.com/alcionai/corso/src/internal/tester"
+	"github.com/alcionai/corso/src/pkg/control"
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+type CollectionSuite struct {
+	tester.Suite
+}
+
+func TestCollectionSuite(t *testing.T) {
+	suite.Run(t, &CollectionSuite{Suite: tester.NewUnitSuite(t)})
+}
+
+func (suite *CollectionSuite) TestReader_Valid() {
+	m := []byte("test message")
+	description := "aFile"
+	ed := &Item{id: description, message: m}
+
+	buf := &bytes.Buffer{}
+	_, err := buf.ReadFrom(ed.ToReader())
+	assert.NoError(suite.T(), err, clues.ToCore(err))
+	assert.Equal(suite.T(), buf.Bytes(), m)
+	assert.Equal(suite.T(), description, ed.ID())
+}
+
+func (suite *CollectionSuite) TestReader_Empty() {
```

Comment:
Negative test case for this can be added.

Reply:
Negative as in, the test reader is not empty?  That would just be the valid case, wouldn't it?